consensus/ibft/src/integration-test/java/tech/pegasys/pantheon/consensus/ibft/support/TestContextBuilder.java

```diff
@@ -115,6 +115,7 @@ public EventMultiplexer getEventMultiplexer() {
   private IbftEventQueue ibftEventQueue = new IbftEventQueue();
   private int validatorCount = 4;
   private int indexOfFirstLocallyProposedBlock = 0; // Meaning first block is from remote peer.
+  private boolean useGossip;
```

Comment:
nit: do we like to explicitly set defaults? (personally I do, but may not be very java-ish)

Reply:
Either way, no strong opinion on this one. Slightly less efficient to have an explicit initialisation and could also be considered redundant. However you could argue it's clearer if it was initialised to false.
Often there is a checkstyle or other code style that prevents from having the explicit value as the default.
So i'm happy to initialise to false as we seemed to have followed that convention elsewhere.